### DISCUSSION TITLE
Using PIL with Hy

### DISCUSSION THREAD

**Discussion by GJ-u** [hylang/hy#2052] [Q&A] [answered]
Hello,
I am working on a small project in Hy and am a little bit confused about the syntax when working with the PIL library.
In Python, I would write:
```python
draw.rectangle((x, y, w, h), fill="black")
```
and i am unsure about how to do the `fill="black"` part in Hy.

**Comment by allison-casey** [comment, marked as answer]
the equivalent would be. Hopefullhy this'll all be more clear when we remake the docs soon!

```clojure
(draw.rectangle (, x y w h) :fill "black")
```

**Reply by GJ-u** [reply]
Thank you very much!
